python/ray/serve/schema.py

```diff
@@ -978,6 +978,73 @@ class ReplicaDetails(ServeActorDetails, frozen=True):
     )
 
 
+@PublicAPI(stability="alpha")
+class MetricsHealth(str, Enum):
+    HEALTHY = "healthy"
+    DELAYED = "delayed"
+    UNAVAILABLE = "unavailable"
+
+
+@PublicAPI(stability="alpha")
+class ScalingSource(str, Enum):
+    DEFAULT = "default"
+    CUSTOM = "custom"
+    EXTERNAL = "external"
+
+
+@PublicAPI(stability="alpha")
+class ScalingStatus(str, Enum):
+    SCALING_UP = "scaling_up"
+    SCALING_DOWN = "scaling_down"
+    STABLE = "stable"
+
+
+@PublicAPI(stability="alpha")
+class ScalingDecision(BaseModel):
+    """One autoscaling decision with minimal provenance."""
+
+    timestamp_s: float = Field(
+        ..., description="Unix time (seconds) when the decision was made."
+    )
+    source: ScalingSource = Field(
+        ..., description="Who made this decision: default/custom/external."
+    )
+    reason: str = Field(
+        ..., description="Short, human-readable reason for the decision."
+    )
+    from_replicas: int = Field(
+        ..., ge=0, description="Replica count before the decision."
+    )
+    to_replicas: int = Field(..., ge=0, description="Replica count after the decision.")
+    policy: Optional[str] = Field(
+        None, description="Policy name or identifier (if applicable)."
+    )
+
+
+@PublicAPI(stability="alpha")
+class DeploymentAutoscalingDetail(BaseModel):
+    """Deployment-level autoscaler observability."""
+
+    class Config:
+        extra = Extra.forbid
+
+    scaling_status: ScalingStatus = Field(
```

Comment:
let's make this aligned with the statuses "upscaling" and "downscaling" https://github.com/ray-project/ray/blob/1373fd20d3df920fc237281496c5abe4c452d3e8/python/ray/serve/_private/common.py#L123-L124